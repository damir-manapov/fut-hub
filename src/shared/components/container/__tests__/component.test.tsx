// import * as React from "react";
// import { Container } from "../component";
// import { shallow } from "enzyme";

// describe("Container", () => {
//   it("должен рендериться", () => {
//     expect(shallow(<Container />)).toMatchSnapshot();
//   });
// });
