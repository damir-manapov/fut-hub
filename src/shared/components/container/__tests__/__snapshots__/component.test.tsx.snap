// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container должен рендериться 1`] = `ShallowWrapper {}`;
